contributing/content-style.md

```diff
@@ -254,7 +254,7 @@ Too many abbreviations can be confusing to readers.
 Make sure to explain any abbreviations that might be unfamiliar the first time they're used.
 
 Avoid abbreviations for Latin terms, such as `i.e.`, `e.g.`, and `etc.`.
-as these aren't always understood correctly.
+as these are always understood correctly.
```

Comment:
Removes a negation that would be correct

Reply:
Yes, it was `aren't` on purpose because those terms are not always understood (they are often misunderstood).
```suggestion
as these aren't always understood correctly.
```

Reply:
Sorry? So this means that the terms like i.e., e.g., and etc. should be avoided?

Or it means that id est, exampli gratia, and etcetera should be used instead of the abbreviated forms?

If this means none of the above two, then do you not think that the negation is wrong?

Please explain. Thanks

Reply:
All Latin terms should be avoided (both the abbreviations and the full forms).

The terms are *not* always understood, which is why we made the suggestion.